tests/test_graph_utils.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import unittest
+import logging
+import rdflib
+from dipper.utils import GraphUtils
+
+logging.basicConfig(level=logging.WARNING)
+logger = logging.getLogger(__name__)
```

Comment:
other ingest all use `LOG = `  because it is short and a GLOBAL

Reply:
fixed